py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/s3_utils.py

```diff
@@ -39,37 +42,24 @@ def invoke_async_lambda(function_arn: str, event: dict) -> None:
                          InvocationType='Event',
                          Payload=json.dumps(event))
 
-def move_3s_objects(obj_list: list[str], 
-                    src_bucket: str,
-                    dest_bucket: str) -> None:
+def move_s3_objects(file_list: list[str],
+                    destination: str) -> None:
```

Comment:
new impl uses pyarrow s3 filesystem to do the moving. no need to know the src bucket.